### ISSUE TITLE
Option to set a configurable delay in seconds when auto playing text-to-speech

### ISSUE THREAD

**Issue by Arkewas** [Peterodox/YUI-Dialogue#74]
Hello,
would it be possible to add a option to set a configurable delay in seconds when Auto Play checkbox is flagged in Accessibility tab ? I'm requesting this feature because most NPC's, when selected, talk for a brief period with their voice so adding a delay option would avoid that the two voices overlap.

**Comment by Arkewas** [comment]
Thank you for implementing this feature so fast in 0.4.8 version, just released. Also cycle through available rewards with TAB, BTWQuests compatibility and Legacy Quest Line tooltip are welcome additions.

**Comment by Peterodox** [comment]
You're welcome!
Unfortunately, since there is no way to tell if the NPC's original speech is complete, we cannot dynamically change the delay's duration, as a result, we'll still experience overlapping issues with NPCs with long voiceovers. You'll have to bear with it 😄
